src/devices/Vl53L1X/Vl53L1X.cs

```diff
@@ -0,0 +1,959 @@
+                i2CDevice.Write(writeArray);
+                Thread.Sleep(10);
+            }
+            catch (IOException ex)
+            {
+                throw new IOException($"Can't change I2C Address to {newAddress}", ex);
+            }
+        }
+
+        private void InitSensor()
+        {
+            for (byte register = 0x2D; register <= 0x87; register++)
+            {
+                WriteRegister(register, _vl53L1XDefaultConfiguration[register - 0x2D]);
+            }
+
+            StartRanging();
+            WaitForDataReady();
+
+            ClearInterrupt();
+            StopRanging();
+            WriteRegister((ushort)Registers.VL53L1X_VHV_CONFIG__TIMEOUT_MACROP_LOOP_BOUND, 0x9); /* two bounds VHV */
+            WriteRegister(0xB, 0); /* start VHV from the previous temperature */
+        }
+
+        /// <summary>
+        /// This function starts the ranging distance operation which is continuous.
+        /// The clear interrupt has to be done after each "get data" to allow the interrupt to be raised when the next data are
+        /// ready.
+        /// 1 = active high (default), 0 =active low.
+        /// If required, use <see cref="InterruptPolarity" /> to change the interrupt polarity.
+        /// </summary>
+        public void StartRanging()
+        {
+            ClearInterrupt();
+            WriteRegister((ushort)Registers.SYSTEM__MODE_START, 0x40);
+            _rangingInitialized = true;
+        }
+
+        /// <summary>
+        /// This function stops the ranging.
+        /// </summary>
+        public void StopRanging()
+        {
+            WriteRegister((ushort)Registers.SYSTEM__MODE_START, 0x00);
+            _rangingInitialized = false;
+        }
+
+        /// <summary>
+        /// This function returns whether to ranging is active or not
+        /// </summary>
+        /// <returns>True if ranging is active. Otherwise false</returns>
+        public bool IsRangingActive
+        {
+            get
+            {
+                byte systemMode = ReadByte((ushort)Registers.SYSTEM__MODE_START);
+                return systemMode == 0x40;
+            }
+        }
+
+        /// <summary>
+        /// Checks if the new ranging data are available by polling the dedicated register.
+        /// </summary>
+        private bool IsDataReady
+        {
+            get
+            {
+                byte temp = ReadByte((byte)Registers.GPIO__TIO_HV_STATUS);
+                return temp != 0;
+            }
+        }
+
+        /// <summary>
+        /// This gets or sets the interrupt polarity. PinValue.High is default.
+        /// </summary>
+        public PinValue InterruptPolarity
+        {
+            get
+            {
+                byte polarity = ReadByte((ushort)Registers.GPIO_HV_MUX__CTRL);
+                polarity = (byte)(polarity & 0x10);
+                return polarity >> 4 == 1 ? PinValue.High : PinValue.Low;
+            }
+            set
```

Comment:
nit, you need an extra CR between } and set. You have that in few other places. Weird that the linter doesn't catch this one.

Reply:
Done